src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -84,6 +85,9 @@ def _print_examples(help_file):
                 _print_indent(u'{0}'.format(e.long_summary), indent)
             _print_indent(u'{0}'.format(e.command), indent)
             print('')
+        indent = 0
+        message = 'For more specific examples, use: az find \'az {}\''.format(help_file.command)
```

Comment:
please use double quotes like `az find "az vm redeploy"`, because single quote won't work on Windows

Reply:
OK I've updated it!